modules/custom/social_language/modules/social_content_translation/config/install/social_content_translation.settings.yml

```diff
@@ -1,6 +0,0 @@
-# TODO: Overrides to these settings are currently unusable (See ContentTranslationDefaultsConfigOverride).
```

Comment:
@nechai I see the point why you decided to remove this config, since it just increases complexity for content translation support, especially when we need to add support for external entities, like courses for example.

But I think we need to get approval for this removal from other teams and make sure they are not used specifically on some projects.

@Kingdutch, @ronaldtebrake could you comment on that? Do you think we can remove this config, and then just enable content translation support for all supported entities when this module is enabled?

Also is it fine to include it in the 10.3 release? Because I am afraid it will be a problem for us if we postpone this to 11.0